Gemfile

```
@@ -5,6 +5,7 @@
 source "https://rubygems.org"
 
 gem 'fastlane'
+gem 'cocoapods', '1.11.3'
```

```suggestion
gem 'cocoapods'
```

Since the version of cocoapods will be set in the `Gemfile.lock` file, I prefer not to set a hard-coded version in this file.

---

this needs the version as there was a mismatch when i ran `bundle install` followed by `pod install` without the version specified since the Podfile.lock specifies the cocoapods version